README.md

```diff
@@ -492,6 +494,41 @@ status:
     status: 'True'
 ```
 
+## Direct Secret Reference
+
+There are scenarios where Provisioned Service is not available.  But there is a `Secret` resource available for binding.  This extension, named Direct Secret Reference, allows Service Binding resource with the service directly pointing to a secret resource.
+
+The `Secret` **SHOULD** contain a `type` entry with a value that identifies the abstract classification of the binding.  It is **RECOMMENDED** that the `Secret` also contain a `provider` entry with a value that identifies the provider of the binding.  The `Secret` **MAY** contain any other entry.
+
+The `.spec.service.kind` attribute **MUST** be `Secret` and the `.spec.service.apiVersion` **MUST** be `v1`.  The `spec.service.name` attribute **MUST** be a `LocalObjectReference`-able to a `Secret`.
```

Comment:
As written, this seems to be saying that all ServiceBindings must directly reference a Secret. What about:

```suggestion
When the `.spec.service.kind` attribute is `Secret` and `.spec.service.apiVersion` is `v1`, the `.spec.service.name` attribute **MUST** be treated as `.status.binding.name` for a Provisioned Service.
```